torch_xla/csrc/runtime/ifrt_computation_client.h

```diff
@@ -253,13 +253,13 @@ class IfrtComputationClient : public ComputationClient {
   struct IfrtComputation : public Computation {
     IfrtComputation(xla::XlaComputation computation,
                     std::vector<std::string> devices,
-                    std::unique_ptr<xla::ifrt::LoadedExecutable> executable)
+                    std::shared_ptr<xla::ifrt::LoadedExecutable> executable)
         : Computation(std::move(computation), std::move(devices)),
           executable(std::move(executable)) {
       output_shardings_ = this->executable->GetOutputShardings();
     }
 
-    std::unique_ptr<xla::ifrt::LoadedExecutable> executable;
+    std::shared_ptr<xla::ifrt::LoadedExecutable> executable;
```

Comment:
While we're here, can we also rename this to executable_ to follow style convention?

Reply:
This is a struct, not a class. The style guide says that struct members don't have trailing _.